skeletons/ini.xml

```diff
@@ -16,18 +16,18 @@
      <entry>&Changelog;</entry>
     </row>
    </thead>
-   <tbody>
```

Comment:
I'm not sure that I agree with this, having related INIs close together is more important than having them alphabetical.